components/inputs/input-time-range.js

```diff
@@ -90,6 +96,62 @@ class InputTimeRange extends RtlMixin(LocalizeCoreElement(LitElement)) {
 		this.enforceTimeIntervals = false;
 		this.labelHidden = false;
 		this.timeInterval = 'thirty';
+
+		this._endDropdownOpened = false;
+		this._endInputId = getUniqueId();
+		this._startDropdownOpened = false;
+		this._startInputId = getUniqueId();
+	}
+
+	get endValue() { return this._endValue; }
+	set endValue(val) {
```

Comment:
I think we'll run into some problems with these checks if the start or end value are valid times but don't match the `timeInterval`. The inner `d2l-input-time` will have the right value and the UI will show the right value but `startValue` or `endValue` will be incorrect.

Reply:
Ah yup that's a good point.